node/src/chain_spec.rs

```diff
@@ -298,18 +298,20 @@ fn mk_genesis(endowed_accounts: Vec<AccountId>, root_key: AccountId, parachain_i
                 .map(|k| (k.0, INITIAL_DHX_DAO_TREASURY_UNLOCKED_RESERVES_BALANCE))
                 .collect(),
         },
-        pallet_collective_Instance1: Default::default(),
-        pallet_membership_Instance1: GeneralCouncilMembershipConfig {
+        general_council: Default::default(),
+        general_council_membership: GeneralCouncilMembershipConfig {
             members: vec![root_key.clone()],
             phantom: Default::default(),
         },
         pallet_treasury: Default::default(),
-        pallet_sudo: SudoConfig {
+        sudo: SudoConfig {
             key: root_key.clone(),
         },
         parachain_info: datahighway_runtime::ParachainInfoConfig {
             parachain_id,
         },
+        aura: Default::default(),
```

Comment:
we're currently using Babe, is it necessary to change to Aura?

Reply:
@ltfschoen As agreed with @jeffstahlnecker and @festelo, I used this branch `/ilya/parachain-update`. As I checked this branch, we ae not using Babe https://github.com/DataHighway-DHX/node/blob/ilya/parachain-update/node/src/chain_spec.rs#L359

After upgrading it to `v0.9.8`, I was getting compilation error. I followed this https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.8/node/src/chain_spec.rs#L157 to fix it.


Please let me know if you want me to remove this and use Babe.

Reply:
I can't see any reason to use Babe. I think we'll be switching to Aura on standalone chain anyway until we resolve whether it causes delays when users query the block header.